components/contribution-flow/FeesOnTopInput.js

```diff
@@ -41,11 +36,11 @@ const getOptions = (amount, currency) => {
       return getOptionFromPercentage(amount, currency, percentage);
     }),
     {
-      label: <FormattedMessage id="NoThankYou" defaultMessage="No, thank you" />,
```

Comment:
We should keep these internationalized. If the issue is that we want to have them as strings, we should use a `defineMessages` + `useIntl()` + `formatMessage`.

Reply:
@Betree : Didn't know about this, sorry. Adjusted the code. Now it should be good. 😄